lib/rules/aria-hidden-body.js

```diff
@@ -0,0 +1,11 @@
+var aria = /^aria-hidden/;
+if (node && node.hasAttributes()) {
```

Comment:
I think we don't need the matches function here since the code is identical to the check. I also don't see where it checks for the body element–perhaps that could go into the `selector` field in the rule json?